concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -5,35 +5,28 @@
 import random
 import time
 from base64                 import b64encode
+from collections import namedtuple
 from freezegun              import freeze_time
 
 from golem_messages         import message
 
 from utils.helpers          import get_current_utc_timestamp
 from utils.testing_helpers  import generate_ecc_key_pair
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
+from api_testing_common import api_request, parse_command_line
```

Comment:
Please follow the convention, import should be aligned to other imports.

Also, single import in single line, alphabetically.